javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/BaseScoring.qll

```diff
@@ -15,69 +15,18 @@ external predicate availableMlModels(
 ATMConfig getCfg() { any() }
 
 /**
- * This module provides functionality that takes an endpoint and provides an entity that encloses that
- * endpoint and is suitable for similarity analysis.
+ * This module provides functionality that takes an endpoint and provides an function that encloses
+ * that endpoint.
  */
-module EndpointToEntity {
+module EndpointToFunction {
   private import CodeToFeatures
 
   /**
-   * Get an entity enclosing the endpoint that is suitable for similarity analysis. In general,
-   * this may associate multiple entities to a single endpoint.
+   * Get a function containing the endpoint that is suitable for featurization. In general,
+   * this associates an endpoint to multiple functions, since there may be more than one multiple entities to a single endpoint.
```

Comment:
"since there may be more than one multiple entities to a single endpoint" doesn't scan well

Reply:
Thanks for pointing out, will fix.